Currently Loaded Modulefiles:
  1) anaconda/default
/PHShome/jjd65/prod_of_experts/spline_learner_poe4d_MM.py:276: RuntimeWarning: overflow encountered in exp
  prob_keep = np.exp(np.sum(num,0) - np.sum(dem,0))
/PHShome/jjd65/prod_of_experts/spline_learner_poe4d_MM.py:314: RuntimeWarning: overflow encountered in exp
  prob_keep = np.exp(np.sum(num, 0) - np.sum(dem, 0))
/PHShome/jjd65/prod_of_experts/spline_learner_poe4d_MM.py:363: FutureWarning: `rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
  mu_new = np.reshape(np.linalg.lstsq(X,g1_aa)[0],(self.num_bugs, self.num_bugs),order='F').T.flatten(order='F')
/PHShome/jjd65/prod_of_experts/spline_learner_poe4d_MM.py:580: RuntimeWarning: More than 20 figures have been opened. Figures created through the pyplot interface (`matplotlib.pyplot.figure`) are retained until explicitly closed and may consume too much memory. (To control this warning, see the rcParam `figure.max_open_warning`).
  fig1, axes1 = plt.subplots(self.num_bugs, self.num_bugs, figsize=(15, 15))
/PHShome/jjd65/prod_of_experts/helper.py:163: RuntimeWarning: More than 20 figures have been opened. Figures created through the pyplot interface (`matplotlib.pyplot.figure`) are retained until explicitly closed and may consume too much memory. (To control this warning, see the rcParam `figure.max_open_warning`).
  num_bugs, 1, sharex=True, figsize=(15, 15))
/PHShome/jjd65/prod_of_experts/helper.py:220: RuntimeWarning: More than 20 figures have been opened. Figures created through the pyplot interface (`matplotlib.pyplot.figure`) are retained until explicitly closed and may consume too much memory. (To control this warning, see the rcParam `figure.max_open_warning`).
  num_bugs, figsize=(15, 5))
/PHShome/jjd65/prod_of_experts/helper.py:236: RuntimeWarning: More than 20 figures have been opened. Figures created through the pyplot interface (`matplotlib.pyplot.figure`) are retained until explicitly closed and may consume too much memory. (To control this warning, see the rcParam `figure.max_open_warning`).
  num_bugs, figsize=(15, 5))
/PHShome/jjd65/prod_of_experts/helper.py:274: RuntimeWarning: More than 20 figures have been opened. Figures created through the pyplot interface (`matplotlib.pyplot.figure`) are retained until explicitly closed and may consume too much memory. (To control this warning, see the rcParam `figure.max_open_warning`).
  num_bugs, figsize=(15, 5))
/PHShome/jjd65/prod_of_experts/helper.py:292: RuntimeWarning: More than 20 figures have been opened. Figures created through the pyplot interface (`matplotlib.pyplot.figure`) are retained until explicitly closed and may consume too much memory. (To control this warning, see the rcParam `figure.max_open_warning`).
  num_bugs, figsize=(15, 5))
Traceback (most recent call last):
  File "./main.py", line 44, in <module>
    # elif args.use_mm and args.aval and args.outdir:
  File "./main.py", line 41, in main
    #                     MEAS_VAR=args.mvar, PROC_VAR=args.pvar, THETA_VAR=args.theta_var, AVAR=args.avar,
  File "/PHShome/jjd65/prod_of_experts/spline_learner_poe4d_MM.py", line 475, in run
    xold, ob=i, proposed_xnew = proposed_xnew, f2 = np.reshape(f2,(self.states.shape[0]-1,self.num_bugs),order='F'),
  File "/PHShome/jjd65/prod_of_experts/helper.py", line 184, in plot_states
    num_bugs = states.shape[1]
  File "/PHShome/jjd65/.conda/envs/dispatcher/lib/python3.7/site-packages/matplotlib/pyplot.py", line 722, in savefig
    res = fig.savefig(*args, **kwargs)
  File "/PHShome/jjd65/.conda/envs/dispatcher/lib/python3.7/site-packages/matplotlib/figure.py", line 2180, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/PHShome/jjd65/.conda/envs/dispatcher/lib/python3.7/site-packages/matplotlib/backend_bases.py", line 2089, in print_figure
    **kwargs)
  File "/PHShome/jjd65/.conda/envs/dispatcher/lib/python3.7/site-packages/matplotlib/backends/backend_agg.py", line 530, in print_png
    cbook.open_file_cm(filename_or_obj, "wb") as fh:
  File "/PHShome/jjd65/.conda/envs/dispatcher/lib/python3.7/contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "/PHShome/jjd65/.conda/envs/dispatcher/lib/python3.7/site-packages/matplotlib/cbook/__init__.py", line 447, in open_file_cm
    fh, opened = to_filehandle(path_or_file, mode, True, encoding)
  File "/PHShome/jjd65/.conda/envs/dispatcher/lib/python3.7/site-packages/matplotlib/cbook/__init__.py", line 432, in to_filehandle
    fh = open(fname, flag, encoding=encoding)
FileNotFoundError: [Errno 2] No such file or directory: 'outdir_new_opt_competing3a_MM0/03-03-2020_08-41-33_states_out.png'
